OpenEdX/Managers/FirebaseAnalyticsManager/FirebaseAnalyticsManager.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  FirebaseAnalyticsManager.swift
+//  OpenEdX
+//
+//  Created by Anton Yarmolenka on 19/02/2024.
+//
+
+import Foundation
+import Firebase
+import Core
+
+class FirebaseAnalyticsManager: AnalyticsService {
```

Comment:
Although we have collaborated on it and name is also fine, but as it's more like a tracker instead of manager, how about renaming it to `FirebaseAnalyticsTracker`.

If it sounds good then the same will go for Segment.

Reply:
What about `FirebaseAnalyticsService`?

Reply:
done